### PR TITLE
LPS-112055

### PR DESCRIPTION
**Pull request by tinatian**

### CODE REVIEW

File: portal-impl/src/com/liferay/portal/security/permission/ResourceActionsImpl.java [hunk 1]
```diff
@@ -470,11 +470,13 @@ public String getPortletRootModelResource(String portletName) {
 
 	@Override
 	public List<String> getResourceActions(String name) {
-		if (name.indexOf(CharPool.PERIOD) != -1) {
-			return getModelResourceActions(name);
+		if (name.indexOf(CharPool.PERIOD) == -1) {
```
Comment (tinatian) [inline portal-impl/src/com/liferay/portal/security/permission/ResourceActionsImpl.java:473]: @dantewang This commit is wrong, please ignore.

Comment (dantewang) [reply portal-impl/src/com/liferay/portal/security/permission/ResourceActionsImpl.java:473]: @tinatian Do you mean the whole commit of https://github.com/dantewang/liferay-portal/pull/1607/commits/7629138bfc3666c380615e37626b0169b8fb6d0b?

Comment (dantewang) [reply portal-impl/src/com/liferay/portal/security/permission/ResourceActionsImpl.java:473]: @tinatian Ah sorry I saw the wrong commit.. It's this one right? https://github.com/dantewang/liferay-portal/pull/1607/commits/c38660ea20f1a58b26f5333a02e66109098ba3a4

Comment (dantewang) [reply portal-impl/src/com/liferay/portal/security/permission/ResourceActionsImpl.java:473]: @tinatian Sorry, I'm confused by Github's UI... Now I see it's commented on https://github.com/dantewang/liferay-portal/commit/7629138bfc3666c380615e37626b0169b8fb6d0b

Comment (tinatian) [reply portal-impl/src/com/liferay/portal/security/permission/ResourceActionsImpl.java:473]: the last one : sf, simplify is wrong. Cause the porlet actions need the checking logic

Comment (dantewang) [reply portal-impl/src/com/liferay/portal/security/permission/ResourceActionsImpl.java:473]: I see! I'll exclude it.

**Comment by dantewang** [comment]
Fetched

**Comment by dantewang** [comment]
Pull request submitted to https://github.com/dantewang/liferay-portal/pull/1612.

**Comment by dantewang** [comment]
Pull request submitted to https://github.com/tinatian/liferay-portal/pull/1827. [See changes here.](https://github.com/dantewang/liferay-portal/compare/c8df0ba18c6d~2...c8df0ba18c6d)

**Pull request closed without merging**